.github/workflows/build-images.yaml

```diff
@@ -109,3 +109,32 @@ jobs:
         run: |
           cd ecosystem/platform/server
           docker buildx bake --progress=plain --push -f ./docker-bake.hcl
+
+  indexer-server:
+    if: ${{ github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'CICD:build-images') }}
```

Comment:
actually I learned recently that in `if` conditions in github actions you can omit the surrounding `${{ }}`, so I suggest to remove it.